emily/handler/src/database/entries/mod.rs

```diff
@@ -255,6 +255,52 @@ pub(crate) trait TableIndexTrait {
                 <<Self::Entry as EntryTrait>::Key as KeyTrait>::PARTITION_KEY_NAME,
             )
             .expression_attribute_values(":v", serde_dynamo::to_attribute_value(partition_key)?)
+            .scan_index_forward(false)
```

Comment:
What are the sort keys for these tables? Maybe we should add a comment explaining this.